fix(module:18n): updating pt_PT translations

drackp2m opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

Add and update translations for pt_PT

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[x] Other... Please describe:
```

## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

wenqi73 commented on components/i18n/languages/pt_PT.ts:
```diff
@@ -45,20 +55,28 @@ export default {
-      nextCentury: 'Próximo século',
-      placeholder: 'Data',
-      rangePlaceholder: ['Data inicial', 'Data final'],
-      monthFormat: 'MMMM'
```
Why delete these translations of datePicker and timePicker?

drackp2m commented:
If you notice I added them in another order. So that the files are consistent with each other and it is easier to compare translations from one file to another. I took care of leaving es_ES, pt_PT and pt_BR in order with respect to en_US.

Sorry for the delay in answering, I have updated my fork with master and everything is still correct. (No new translations have been added or removed in the file `components/i18n/languages/en_US.ts`

wenqi73 commented:
@drackp2m Oh sorry for the misunderstood, but it is better to restore `monthFormat`, even if the en_US does not have it. By the way, remember to rebase master.

drackp2m commented:
> rebase master

Perfect @wenqi73, I did it locally, and since I saw that there were no conflicts, I didn't upload it, but today I corrected that.

---

One last question then: What language, or what file could I use to know which translation keys are really valid? Chinese maybe?

wenqi73 commented:
@drackp2m Still has build errors, you should rebase the remote master of ng-zorro.

wenqi73 commented:
> One last question then: What language, or what file could I use to know which translation keys are really valid? Chinese maybe?

@drackp2m The en_US is the valid one, since we sync these files from [ant-design](https://github.com/ant-design/ant-design/tree/master/components/locale), you will see some keys are not used in ng-zorro, maybe we are not using these keys correctly.
